rosidl_generator_c/test/test_interfaces.c

```diff
@@ -896,3 +904,325 @@ int test_submessages(void)
 
   return 0;
 }
+
+/**
+ * Test message with bounded array nested types
+ */
+int test_bounded_array_nested(void)
+{
+  bool res;
+  size_t size = 4;
```

Comment:
Done in https://github.com/ros2/rosidl/pull/222/commits/4f9033a3d1f78288dbbf27f7adad80e220850b8c thanks for pointing it out